### DISCUSSION TITLE
Can you organise outside collaborators into teams?

### DISCUSSION THREAD

**Discussion by AndrWeisR** [community/community#78092] [New to GitHub] [answered]
I will have a team of developers from an external organisation working on some of my repos. I want to invite the external vendors as outside collaborators. Is it possible to organise a number of such outside collaborators into a team? Or can only organisation members be organised into teams?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @AndrWeisR, 

Only organization members can be members of a team. Thanks for asking this, I am going to update our docs to make this more clear. 

Cheers, 
Lili

**Reply by Tonylov1** [reply]
How do you become a member. I?

**Reply by xh6nvpfy7v-stack** [reply]
Musím tě povolit.

**Comment by anasiede** [comment]
Hello, I am looking for a remote job online

**Comment by queenofcorgis** [comment]
@anasiede, we're happy you're here 😀

However, we currently do not allow job listing or seeking on the Community Discussions as a result your comment has been hidden. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍

**Comment by A223456789010304050** [comment]
Україна 
1G50G 1GDELETE 🟡🌍🔵
